### PR TITLE
Make the appInfo exported and vendor external dependencies

### PR DESCRIPTION
**Pull request by bzon**
- The `appInfo` should be exported as it is an important type that can be reused for projects that want to use this package.
- Let us also vendor the dependencies to have more control in this package stability.

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+			"revisionTime": "2017-11-25T03:40:40Z"
+		},
+		{
+			"checksumSHA1": "RXqSQ/p78DgPX/UxRF8fjJPcMdg=",
+			"path": "github.com/shogo82148/androidbinary/apk",
+			"revision": "f2570763f91ec28d97aa329a78bfb2b93eb41822",
+			"revisionTime": "2017-11-25T03:40:40Z"
+		}
+	],
+	"rootPath": "github.com/bzon/ipapk"
```
Comment (phinexdaz) [inline vendor/vendor.json]: The "rootPath" field in the bottom of vendor.json file has any influence?

Comment (bzon) [reply vendor/vendor.json]: Hmm, I just noticed. It shouldn't be, I changed it now to your namespace and did a test without an issue.

File: vendor/vendor.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+			"revisionTime": "2017-11-25T03:40:40Z"
+		},
+		{
+			"checksumSHA1": "RXqSQ/p78DgPX/UxRF8fjJPcMdg=",
+			"path": "github.com/shogo82148/androidbinary/apk",
+			"revision": "f2570763f91ec28d97aa329a78bfb2b93eb41822",
+			"revisionTime": "2017-11-25T03:40:40Z"
+		}
+	],
+	"rootPath": "github.com/phinexdaz/ipapk"
```
Comment (bzon) [inline vendor/vendor.json]: @phinexdaz I changed it to your namespace now. I've done a test and had no issue.

Comment (phinexdaz) [reply vendor/vendor.json]: ok, thanks for your contribution

**Pull request merged**